### ISSUE TITLE
Thermal Spectrum Reactors

### ISSUE THREAD

**Issue by john-science** [terrapower/armicontrib-openmc#19]
ARMI has a sample thermal spectrum reactor (with a square lattice) here:

https://github.com/terrapower/armi/blob/872e2dac316d937d06a9d3a7a314c48075204a2b/armi/utils/reportPlotting.py#L732

What do you think?  Could we support that example with the current OpenMC benchmark?

**Comment by john-science** [comment]
It looks like the above C5G7 reactor has already been put through OpenMC:  https://docs.openmc.org/en/v0.9.0/examples/mg-mode-part-i.html

So, that's helpful, for comparison.

**Comment by john-science** [comment]
@AidanMcDonald 

This could even be worth published, to show that ARMI can reproduce these results!

**Comment by AidanMcDonald** [comment]
Yes, we can absolutely support the C5G7. I just pushed an entry point that runs it, but I still need to add postprocessing and verify against the existing run.
